File: 260127/짝수의 합 n번 구하기/find-the-sum-of-even-numbers-n-times.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int N=sc.nextInt();
        int[] arr=new int[2];
        int answer;

        int i=0;

        while(i<N){
            answer=0;
            for(int a=0; a<arr.length; a++){
                arr[a]=sc.nextInt();
            }
            for(int j=arr[0]; j<=arr[1]; j++){
                if(j%2==0){
                    answer+=j;
                }
            }
            System.out.println(answer);
            i++;
        }
    }
}

File: 260304/짝수의 합 n번 구하기/find-the-sum-of-even-numbers-n-times.cpp
#include <iostream>
using namespace std;


void sum(int a, int b){
    int add=0;
    for(int i=a; i<b+1; i++){
        i%2==0?add+=i:false;
    }
    cout<<add<<endl;
}

int main() {
    // Please write your code here.
    int n;
    cin>>n;

    for(int i=0; i<n; i++){
        int a,b;
        cin>>a>>b;

        sum(a,b);
    }
    return 0;
}